managedtenants/utils/schema.py

```diff
@@ -35,7 +35,7 @@ def load_draft7_schema(path_or_file):
     """
     if isinstance(path_or_file, Path):
         f = file_to_close = open(  # pylint: disable=consider-using-with
-            path_or_file, "r"
+            path_or_file, encoding="utf8"
```

Comment:
Was the file access mode removed by accident? Could you add it back?

Reply:
Ohh yes, thanks for catching it, my bad!